tslint.json

```diff
@@ -70,7 +70,7 @@
         "no-regex-spaces": true,
         "no-sparse-arrays": true,
         "no-stateless-class": true,
-        // TODO: "no-string-literal": true,
+        "no-string-literal": false, // changed
```

Comment:
I'm not sure I understand why `foo['bar']` would ever be better than `foo.bar`? Don't you lose a bunch of useful things like autocomplete and type safety?